asciidoc/workloads/azure/backend/java/setting_up_cosmos_db_locally_java.adoc

```diff
@@ -93,7 +102,11 @@ Now running the below should list 4 certificates.
 keytool -list -keystore cacerts
 ----
 
-=== Unix
+====
+
+.Unix
+[%collapsible]
+====
 
 ==== Download the certificate
```

Comment:
Need to add [discrete] above any headers which are set in collapsible blocks, otherwise they don't render correctly.

Reply:
fixed other docs that i have not added [discrete]